lib/led/rgb.js

```diff
@@ -215,6 +215,7 @@ class RGB {
         }
       },
       update: {
+        writable: true,
```

Comment:
Is this needed for testing?

Reply:
Yes - within [this test](https://github.com/code-dot-org/johnny-five/blob/e203c5873af3925b92177a3a442fd7b7ed8df412/test/rgb.js#L820) in `test/rgb.js`